redirects.yml

```diff
@@ -0,0 +1,168 @@
+  to_url: https://docs.nextstrain.org/projects/ncov/en/latest/advanced/naming_clades.html
+
+- type: page
+  from_url: /tutorials/SARS-CoV-2/steps/customizing-visualization.html
+  to_url: https://docs.nextstrain.org/projects/ncov/en/latest/analysis/customizing-visualization.html
+
+- type: page
+  from_url: /tutorials/SARS-CoV-2/steps/customizing-analysis.html
+  to_url: https://docs.nextstrain.org/projects/ncov/en/latest/analysis/customizing-analysis.html
+
+- type: page
+  from_url: /tutorials/SARS-CoV-2/steps/running.html
+  to_url: https://docs.nextstrain.org/projects/ncov/en/latest/analysis/running.html
+
+- type: page
+  from_url: /tutorials/SARS-CoV-2/steps/orientation-files.html
+  to_url: https://docs.nextstrain.org/projects/ncov/en/latest/analysis/orientation-files.html
+
+- type: page
+  from_url: /tutorials/SARS-CoV-2/steps/orientation-workflow.html
+  to_url: https://docs.nextstrain.org/projects/ncov/en/latest/analysis/orientation-workflow.html
+
+- type: page
+  from_url: /tutorials/SARS-CoV-2/steps/data-prep.html
+  to_url: https://docs.nextstrain.org/projects/ncov/en/latest/analysis/data-prep.html
+
+- type: page
+  from_url: /tutorials/SARS-CoV-2/steps/setup.html
+  to_url: https://docs.nextstrain.org/projects/ncov/en/latest/analysis/setup.html
+
+- type: page
+  from_url: /tutorials/SARS-CoV-2/steps/index.html
+  to_url: https://docs.nextstrain.org/projects/ncov/en/latest/index.html
+
+- type: page
+  from_url: /tutorials/SARS-CoV-2/index.html
+  to_url: https://docs.nextstrain.org/projects/ncov/en/latest/index.html
+
+- type: page
+  from_url: /guides/install/cli-install.html
+  to_url: https://docs.nextstrain.org/projects/cli/en/latest/installation/
+
+- type: page
+  from_url: /guides/install/auspice-install.html
+  to_url: https://docs.nextstrain.org/projects/auspice/en/stable/introduction/install.html
+
+- type: page
+  from_url: /guides/install/augur_install.html
+  to_url: https://docs.nextstrain.org/projects/augur/en/stable/installation/installation.html
+
+- type: page
+  from_url: /guides/install/auspice-install
+  to_url: https://docs.nextstrain.org/projects/auspice/en/stable/introduction/install.html
+
+- type: page
+  from_url: /guides/install/windows-help.html
+  to_url: install-nextstrain.html
+
+- type: page
+  from_url: /guides/install/local-installation.html
+  to_url: install-nextstrain.html
+
+- type: page
+  from_url: /guides/install/index.html
+  to_url: install-nextstrain.html
+
+- type: page
+  from_url: /guides/install/
+  to_url: install-nextstrain.html
+
+- type: page
+  from_url: /tutorials/zika_tutorial.html
+  to_url: /tutorials/zika.html
+
+- type: page
+  from_url: /tutorials/quickstart.html
+  to_url: /tutorials/running-a-workflow.html
+
+- type: page
+  from_url: /tutorials/zika.html
+  to_url: /tutorials/creating-a-workflow.html
+
+- type: page
+  from_url: /tutorials/tb_tutorial.html
+  to_url: /tutorials/creating-a-bacterial-pathogen-workflow.html
```

Comment:
If you remove these, we can merge this PR before #84, then I can add them in there.

Reply:
Sure.

Note that it is safe to have redirects defined for pages that haven't yet moved, because RTD only consult page redirects when handling a 404 (our readme [discusses this](https://github.com/nextstrain/docs.nextstrain.org#configuring-redirects) a little).

Reply:
Done.

Reply:
@tsibley oh cool, well this will make for better git commit history at the least. Thanks!